wire-ios-transport/Source/Background/BackgroundActivityFactory.swift

```diff
@@ -243,25 +241,21 @@ private let zmLog = ZMSLog(tag: "background-activity")
         if let currentBackgroundTask = self.currentBackgroundTask {
             if let activityManager = activityManager {
                 let value = SafeValueForLogging(currentBackgroundTask.rawValue)
-                zmLog.safePublic("Finishing background task: \(value)")
+                WireLogger.backgroundActivity.info("Finishing background task: \(value)")
                 // We might get killed pretty soon, let's flush the logs
                 ZMSLog.sync()
                 activityManager.endBackgroundTask(currentBackgroundTask)
             } else {
-                zmLog.safePublic("Finishing background task: failed, activityManager is nil")
+                WireLogger.backgroundActivity.info("Finishing background task: failed, activityManager is nil")
             }
             self.currentBackgroundTask = nil
         } else {
-            zmLog.safePublic("Finishing background task: no current background task")
+            WireLogger.backgroundActivity.info("Finishing background task: no current background task")
```

Comment:
we don't have the same level here, so it won't make prod builds. add `,attributes: .safePublic)` to keep the same behaviour.

Or was it intended?

Reply:
@David-Henner I took this from you, can you please help me?

Reply:
thanks, updated with f5768f5c6eae249bcae0bfa999c3737e35ac93a0